big_tests/src/ct_mongoose_hook.erl

```diff
@@ -134,14 +134,15 @@ check_server_purity(Suite, Config) ->
 
 do_check_server_purity(_Suite) ->
     Funs = [fun check_sessions/0,
-        fun check_registered_users/0,
-        fun check_registered_users_count/0,
-        fun check_offline_messages/0,
-        fun check_active_users/0,
-        fun check_privacy/0,
-        fun check_private/0,
-        fun check_vcard/0,
-        fun check_roster/0],
+            fun check_registered_users/0,
+            fun check_registered_users_count/0,
+            fun check_offline_messages/0,
+            fun check_active_users/0,
+            fun check_privacy/0,
+            fun check_private/0,
+            fun check_vcard/0,
+            fun check_roster/0,
+            fun check_carboncopy/0],
```

Comment:
@chrzaszcz I've just realised, this is checking an ets table called `carboncopy` that doesn't really exists in mim 🤔